content/en/synthetics/mobile_app_testing/_index.md

```diff
@@ -36,26 +36,30 @@ You can create mobile app tests in Datadog by navigating to [**Digital Experienc
 
 {{< img src="mobile_app_testing/new_test_2.png" alt="Create a Synthetic Mobile Test" style="width:50%;">}}
 
-### Flakiness 
+## Configuration
 
-Flakiness is a pain point in end-to-end testing. Test failures are occasionally caused by valid frontend code changes that impact an identifier, not by an actual application issue.
+You may create a test using one of the following options:
 
-To prevent flaky tests, Datadog uses an algorithm that leverages a set of locators to target elements in mobile app tests. A small change in the UI may modify an element (for example, moving it to another location). The mobile app test automatically locates the element again based on points of reference that are not affected by the change. 
+- **Create a test from a template**:
 
-When the test runs successfully, the mobile app test recomputes (or "self heals") any broken locators with updated values. This ensures your tests do not break from simple UI updates and your tests are automatically adapting to your mobile application's UI. 
+    1. Hover over one of the following pre-populated templates and click **View Template**. This opens a side panel displaying pre-populated configuration information, including: Test Details, Request Details, Alert Conditions, and Steps.<br /><br>
 
-## Configuration
+       {{< img src="/synthetics/browser_tests/synthetics_templates_mobile.mp4" alt="Video of Mobile Application test landing page with templates" video="true" >}}
+
+    2. Click **+Create Test** to open the configuration page, where you can review and edit the pre-populated configuration options. The fields presented are identical to those available when creating a test from scratch.
+    3. Click **Save & Quit** in the upper right hand corner when you are ready to submit your Mobile Application Test.<br /><br>
```

Comment:
Same button description thing :) Also not sure why "Mobile Application Test" is capitalized throughout this topic? Again, that's something that can wait for another time.

Reply:
same thing as Browser Test as it's the name of the product